src/Groups/GroupDetails/GroupInformation/GroupInformation.js

```diff
@@ -0,0 +1,143 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Row,
+  Col,
+  KeyValue,
+} from '@folio/stripes/components';
+import { ViewMetaData } from '@folio/stripes/smart-components';
+import {
+  AcqUnitsView,
+  AmountWithCurrencyField,
+} from '@folio/stripes-acq-components';
+
+const GroupInformation = ({
+  metadata,
+  name,
+  code,
+  status,
+  description,
+  acqUnitIds,
+  fiscalYears,
+}) => {
+  return (
+    <Fragment>
+      {
+        metadata && <ViewMetaData metadata={metadata} />
+      }
+
+      <Row>
+        <Col
+          data-test-group-information-name
+          xs={3}
+        >
+          <KeyValue
+            label={<FormattedMessage id="ui-finance.groups.item.information.name" />}
+            value={name}
+          />
+        </Col>
+
+        <Col
+          data-test-group-information-code
+          xs={3}
+        >
+          <KeyValue
+            label={<FormattedMessage id="ui-finance.groups.item.information.code" />}
+            value={code}
+          />
+        </Col>
+
+        <Col
+          data-test-group-information-fiscal-year
+          xs={3}
+        >
+          <KeyValue
+            label={<FormattedMessage id="ui-finance.groups.item.information.fiscalYear" />}
+            value={fiscalYears}
+          />
+        </Col>
+
+        <Col
+          data-test-group-information-status
+          xs={3}
+        >
+          <KeyValue
+            label={<FormattedMessage id="ui-finance.groups.item.information.status" />}
+            value={status}
```

Comment:
I don't think you need to put `value` if there is a children

Reply:
yep, should be removed